### DISCUSSION TITLE
Temps freezing - no increase or decrease

### DISCUSSION THREAD

**Discussion by thedimou** [artisan-roaster-scope/artisan#925] [General]
Hi there,
Has anyone encountered a problem with temps when using a screen with a pc? My main built (arduino uno + 3 max31856 with k type sensors, tc4 protocol) was configured with a laptop and now I want to install it in a pc. I have made all the needed settings but it shows wrong temps.
The first one seems ok, the second about 10 Celsius above the first while the third is about 20 Celsius above the first and stack, no ups or downs.
Then, I plug in the laptop with its integrated screen and everything is normal, about 2-3 degrees difference, as the machine is not working I get room temps. I plug in the screen on the laptop and I get the same false readings.
I have used several cables like hdmi, dvi, vga and I get the same issue. I even used different screens.
Any ideas?

**Comment by MAKOMO** [comment]
This seems not an Artisan issue, but rather a grounding issue. I'll transfer this one to a discussion.
